src/FSharp.Core/list.fsi

```diff
@@ -1288,6 +1444,9 @@ module List =
     val tryLast: list:'T list -> 'T option
 
     /// <summary>Builds a new collection whose elements are the results of applying the given function
+    /// 
+    /// Time complexity: O(n) - where n is the list length
+    /// Space complexity: O(n) - for the new list
```

Comment:
Here the added complexity information should be included at the end of the summary and not in the middle of a sentence